nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/queue/clustered/server/ConnectionLoadBalanceServer.java

```diff
@@ -169,8 +169,10 @@ public void run() {
                         break;
                     }
                 } catch (final Exception e) {
+                    stopped = true;
                     if (socket != null) {
                         try {
+                            logger.debug("closing socket");
```

Comment:
This logging statement should include more details about the connection.

Reply:
removed (leftover debugging)